### PR TITLE
Fixed bug

### PR DESCRIPTION
**Pull request by kkrampa**
@czue

### CODE REVIEW

**Comment by dimagimon** [comment]
:+1: No lint errors found.

File: custom/ewsghana/reports/maps.py [hunk 1]
```diff
@@ -47,7 +46,11 @@ def _get_icon_and_color(self, value):
         }[value]
 
     def get_data(self):
-        for loc in self.locations:
+        filtered_locations = [
+            location for location in self.locations
+            if SQLProduct.objects.get(product_id=self.active_product.get_id) in location.sql_location.products
```
Comment (czue) [inline custom/ewsghana/reports/maps.py]: it looks like this db lookup will be called once per location - can you pull it out of the list comprehension?

**Comment by czue** [comment]
can you clarify how this addresses http://manage.dimagi.com/?157858?

**Comment by kkrampa** [comment]
@czue sorry bad case id:
This PR is related to: http://manage.dimagi.com/default.asp?157859#876948

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by kkrampa** [comment]
@czue

**Pull request merged**